configs/dictionaries/essixpointseven_percentages.yaml

```diff
@@ -0,0 +1,4 @@
+---
+A: 1
+B: 1
+Z: 98
```

Comment:
Should this be:

```
A: 50
B: 50
```
with no Z? Can chat off-thread

Reply:
Thanks! Simon explained I need to remove the Z as it is only needed in the rendering app config, not the CDN config because A or B need to be assigned by the CDN.